### PR TITLE
add string.py with constants

### PR DESCRIPTION
**Pull request by wolfv**
These are a few constants copied from `cpython` `Lib/string.py`.

### CODE REVIEW

**Comment by certik** [comment]
I think this looks great. Let's add some tests to ensure it actually works?

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -532,6 +532,7 @@ RUN(NAME test_str_02         LABELS cpython llvm llvm_jit c)
 RUN(NAME test_str_03         LABELS cpython llvm llvm_jit c)
 RUN(NAME test_str_04         LABELS cpython llvm llvm_jit c wasm)
 RUN(NAME test_str_05         LABELS cpython llvm llvm_jit c)
+RUN(NAME test_string_01      LABELS cpython llvm llvm_jit c)
```
Comment (certik) [inline integration_tests/CMakeLists.txt:535]: Either rename this to `test_string`, or (rather) rename the file `test_string.py` to `test_string_01.py`.

**Review by certik — APPROVED** [review]

**Review by certik — APPROVED** [review]
I think this is good, thanks!

File: integration_tests/test_string_01.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from string import ascii_lowercase, ascii_letters
+
+def test_string():
```
Comment (namannimmo10) [inline integration_tests/test_string_01.py:3]: You need to call the function below ;)

Comment (wolfv) [reply integration_tests/test_string_01.py:3]: ouch!

Comment (wolfv) [reply integration_tests/test_string_01.py:3]: fixed in #2788

Comment (namannimmo10) [reply integration_tests/test_string_01.py:3]: Perhaps one can write a custom flake8 plugin using https://pypi.org/project/flake8-plugin-utils/ that checks all functions that are defined, are being called.

Comment (wolfv) [reply integration_tests/test_string_01.py:3]: I don't think it has to be custom. You could probably use `ruff`, too :)

Comment (certik) [reply integration_tests/test_string_01.py:3]: LPython itself should give a warning for something like this.

**Pull request merged**